test/util/test_runners/integrations_tests_runner.py

```diff
@@ -0,0 +1,136 @@
+"""
+This is a runner module for all integration tests.
+The purpose of this file is to be a cross platform tests runner
+for all integration tests
+
+Roadmap:
+1) run all integration unitests from this file
```

Comment:
all integration tests
i know unittest is the name of the library we work with but integration unittest in the same sentence is confusing

Reply:
Changed. You are right I need to start think of them as not unitests